components/collection/unlockable/UnlockableContainer.vue

```diff
@@ -159,6 +159,7 @@ import { OSlider } from '@oruga-ui/oruga'
 import { timeAgo } from '@/components/collection/utils/timeAgo'
 import { collectionId, countDownTime } from './const'
 import { UNLOCKABLE_CAMPAIGN, createUnlockableMetadata } from './utils'
+import { endOfHour, startOfHour } from 'date-fns'
```

Comment:
- https://youmightnotneed.com/date-fns#endOfHour
- https://youmightnotneed.com/date-fns#startOfHour

Reply:
Quick hack sir + we already use date-fns a lot